src/mame/drivers/mc10.cpp

```diff
@@ -247,15 +257,15 @@ DRIVER_INIT_MEMBER(mc10_state,mc10)
 	/* initialize memory */
 	m_ram_base = m_ram->pointer();
 	m_ram_size = m_ram->size();
-	m_pr_state = PRINTER_WAIT;
+	m_pr_state = printer_state::WAIT;
```

Comment:
This is not a constant configuration value, but it isn't saved.  Fix it or remove the flag indicating save state support.  Same with `m_pr_counter` (which incidentally isn't being initialised here - under what circumstances is it valid?).